Documentation/release-notes/README.md

```diff
@@ -0,0 +1,177 @@
+the following settings are enabled:
+
+  * **Android Package Format** set to **bundle**
+  * **Code shrinker**
+  * **AOT Compilation**
+  * **Enable Startup Tracing**
+  * **Linking**
+
+Be sure to disable these settings for the best Debug configuration deployment
+performance.
+````
+
+### External tool or library version update
+
+Example:
+
+```
+#### Mono.Data.Sqlite SQLite version update
+
+The version of SQLite used by Mono.Data.Sqlite in Xamarin.Android has been
+updated from 3.27.1 to [3.28.0](https://sqlite.org/releaselog/3_28_0.html),
+bringing in several improvements and bug fixes.
+```
+
+### Other issue fix or small improvement
+
+  * Use past tense verbs when describing the previous problematic behavior.
+  * List all public GitHub or Developer Community items that were fixed by the
+    change.  If no public items were fixed, or if the scope of the fix was
+    larger than any of the individual items, list the PR instead.
+  * Where possible and appropriate, include the error text that users would have
+    seen before the fix.
+  * If the fix is for a regression and you know when the problem started,
+    include that version info.
+
+Examples:
+
+```
+  * [Developer Community 743965](https://developercommunity.visualstudio.com/content/problem/743965/newtonsoftjsonjsonreaderexception-unexpected-chara.html),
+    [GitHub 3626](https://github.com/xamarin/xamarin-android/issues/3626):
+    Starting in Xamarin.Android 10.0, *Newtonsoft.Json.JsonReaderException:
+    Unexpected character encountered* caused `JsonConvert.DeserializeObject()`
+    to fail in apps built in the Release configuration.
+
+  * [GitHub 3498](https://github.com/xamarin/xamarin-android/issues/3498):
+    Writing to the `System.IO.Stream` returned by
+    `Application.Context.ContentResolver.OpenOutputStream()` completed without
+    error but did not write any bytes to the output location.
+
+  * [GitHub PR 3561](https://github.com/xamarin/xamarin-android/pull/3561):
+    Starting in Xamarin.Android 10.0, the `_GenerateJavaStubs` target could run
+    during builds where it wasn't needed if any of the previous builds had
+    involved a change to a class that inherited from `Java.Lang.Object`.
+
+  * [GitHub PR 3685](https://github.com/xamarin/xamarin-android/pull/3685):
+    *warning ANDJS0000* was always shown for the `jarsigner` warnings *The
+    signer's certificate is self-signed.* and *No -tsa or -tsacert is provided
+    and this jar is not timestamped.*  Those messages were not relevant for
+    Xamarin.Android users, so Xamarin.Android 10.1 now reports them as
+    informational messages rather than warnings.
+```
+
+### Submodule or `.external` bump
+
+  * Use a single file for all of the notes for the bump.
+  * Use section headings to help organize the notes.
+  * For items that do not have a public issue or PR on GitHub or Developer
+    Community, use the xamarin-android bump PR as the item link instead.
```

Comment:
Like in the *Other issue fix or small improvement* section,

```suggestion
    Community, you can skip adding any links to the item.
```

There won't be a PR number yet in cases where the draft release notes are committed to the author's branch before a PR is even opened.